charts/kube-prometheus-stack/hack/update_mixins.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/bash
```

Comment:
Detect bash path and add error control (fail on error, unbounded variable and fail on piped process).

```suggestion
#!/usr/bin/env bash

set -euo pipefail
```